gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -154,7 +153,7 @@ const changeIssuedToUnissuedFacility = async (req, res) => {
  * validates first and gets parameters {Object} from validation function
  * displays success message and redirects to unissued facilities list
  * @param {req}
- * @returns {res}
+ * @param {res}
```

Comment:
Can we also please add `@returns`

Reply:
This docstring seems not to be correct as `req` is not a type but is being used as the type in this jsdoc. Could we update this to instead be `@param {object} req` or `@param {import('express').Request} req`? In terms of the return type, this function seems to return either a `res.redirect` or `res.render` which are both functions which return `void`, so not including the `@returns` tag is semantically correct as returning `void` is the equivalent of not returning anything. However, if we want to include the `returns` tag, then please could it be `@returns {void}`

Reply:
I'm not updating the doc strings here for these, as most of `res` `req` are also wrong in the same way, and it'll be it's own ticket to bring all of these into being correct (this ticket is addressing the issues regarding promises).

For the sake of completionism, I've added a return as promise void.

Reply:
Agree with @JoshBinns2000 on `@param {object} req` , however it will be a separate PR and needs to be scoped in @AlexBramhill  could you please create a Jira ticket pertinent to the JSDoc please and then we can close this PR. Thanks

Reply:
Will add a ticket